package.json

```diff
@@ -2,6 +2,7 @@
   "name": "react-formal",
   "version": "0.17.9",
   "description": "Classy HTML form management for React",
+  "jsnext:main": "src/index.js",
```

Comment:
not sure what you mean about going out of your way to include this with webpack though... are you saying you can't just require it? you should have to configure anything to use this.